File: assignments/mario.c
# include <stdio.h>
# include <cs50.h>



int main(void)

{
    int height, level, width, space;
    do
    {
        // ask user how tall the pyramid should be and create the pyramid
        height = get_int("How many levels would you like your pyramid to be?: ");
    }
    //do while loop required >0 and <9
    while (height < 1 || height > 8);
    
    for (level = 0; level < height; level++)

    {
        // add spaces 
        for (space = 0; space < height - level - 1; space++)
        {
            printf(" ");
        }
        // add hashes
        for (width = 0; width <= level; width++)
        {
            printf("#");
        }
        
        printf(" ");
        printf(" ");
        
        
        // add hashes
        for (width = 0; width <= level; width++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: assignments/pset6/mario.py
from cs50 import get_int


def main():
    # prompt the user again if the height is < 1 or > 8 
    height = get_int("Please select the height of your pyramid:\n")
    while height < 1 or height > 8:
        height = get_int("Please select the height of your pyramid:\n")
            
    for i in range(height):
        space(height, i)
        hash(i)
        print("  ", end="")
        hash(i)
        print()
        
            
def space(height, i):
    for j in range(height, i + 1, -1):
        print(" ", end="")
    
            
def hash(i):
    for k in range(0, i + 1):
        print("#", end="")
  
            
main()
            
        